mirage/factories/file-report.js

```diff
@@ -1,15 +1,29 @@
+import { isEmpty } from '@ember/utils';
+import dayjs from 'dayjs';
 import { Factory } from 'ember-cli-mirage';
 import faker from 'faker';
 
 export default Factory.extend({
-  generatedOn: faker.date.past(),
+  generated_on: faker.date.past(),
   language: faker.random.arrayElement(['en', 'ja']),
   format: faker.random.arrayElement(['DEFAULT', 'DETAILED']),
   progress: faker.random.number({ min: -1, max: 100 }),
-  isGenerating() {
+
+  is_generating() {
     return this.progress >= 0 && this.progress <= 99;
   },
-  isGenerated() {
+
+  is_generated() {
     return this.progress == 100;
   },
+
+  generated_on_date_time() {
```

Comment:
this is already defined in model so you can get the model from store for value reference

Reply:
When using just mirage generated models, this becomes required. I just tried it out.